### PR TITLE
Issue/1325 if statement

### PR DESCRIPTION
**Pull request by sanderr**

### CODE REVIEW

**Comment by sanderr** [comment]
added recommended scope tests

**Comment by arnaudsjs** [comment]
We should add two more things here:

1. Documentation in the Inmanta language reference (see: docs/language.rst)
2. An entry in CHANGELOG.md

**Comment by bartv** [comment]
Documentation is also related to #855 
This is one more use case for conditions/expressions

**Comment by sanderr** [comment]
Could someone provide me with a useful example for the documentation, or replace the toy example I added? I have added the explanation and the syntax, but I am currently lacking practical application knowledge.

File: docs/language.rst [hunk 1]
```diff
@@ -432,6 +434,25 @@ The syntax is:
...
 
 
+If statement
+============
+
+An if statement allows to branch on a condition.
+
+.. code-block:: inmanta
+
+    if 1 == 1:
```
Comment (wouterdb) [inline docs/language.rst]:
```
if nodecount > 1:
  self.cluster_mode = True
else:
  self.cluster_mode = False
end

**Pull request merged**